source/_integrations/zwave_js.markdown

```diff
@@ -638,11 +638,15 @@ You can, but you cannot run them both at the same time. Only one of them can be
 
 Switching does not require renaming your devices.
 
-1. Note your network security keys from your current add-on, then uninstall that add-on. **Do not remove the Z-Wave JS integration or you will lose all device and entity naming.**
+1. Disable the Z-Wave JS integration. **Do not remove the Z-Wave JS integration or you will lose all device and entity naming.** This will automatically stop the official Z-Wave JS add-on.
 
-2. Install and configure the new add-on, including setting those network security keys.
+2. Note your network security keys from the official add-on, then uninstall that add-on.
```

Comment:
We shouldn't uninstall the official add-on until we've updated the config entry by going through the config flow again.